schnorr_fun/src/frost_backup.rs

```diff
@@ -0,0 +1,198 @@
+//! Backup scheme for FROST secret shares
+//!
+//! # Description
+//!
+//! Based on https://bips.xyz/93
+//!
+//! ## Backup format (bech32 chars)
+//!
+//! human readable: "fr"            // (2)
```

Comment:
I agree with the premise but think using `fr` to indicate a frost backup is cooler to users

SSS plebs can use frost labelled backups and reconstitute their seed if they must 😄

Reply:
Is our specification going to be specifically for frost backups, or a wider shamir secret sharing backup spec? 

If we don't capture a wider SSS specification here and one was made in the future, it will likely be able to say `fr` data is compatible with `sss`.

And if it wasn't compatible, then this implies a wider SSS backup spec needs data that we are missing currently. I don't think there is anything extra needed, but we ourselves are not using SSS for anything, and maybe someone wants to do something clever in the future.

`sss` is also longer, and `ss` is no good

Reply:
changed to `frost`